### PR TITLE
feat(sealed-secrets): add sealed-secrets model

### PR DESCRIPTION
**Pull request by douglasduteil**
Add [Sealed Secrets](https://github.com/bitnami-labs/sealed-secrets)

### CODE REVIEW

File: packages/sealed-secrets/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Install with npm.
+
+```sh
+npm install @kubernetes-models/sealed-secrets
+```
+
+## Usage
+
+```js
+import { BackendConfig } from "@kubernetes-models/sealed-secrets/gen/bitnami.com/v1alpha1/SealedSecret";
```
Comment (tommy351) [inline packages/sealed-secrets/README.md:16]:
```suggestion
import { SealedSecret } from "@kubernetes-models/sealed-secrets/gen/bitnami.com/v1alpha1/SealedSecret";
```

File: packages/sealed-secrets/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@kubernetes-models/sealed-secrets",
+  "version": "0.1.1",
+  "description": "Kubernetes sealed-secrets models",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tommy351/kubernetes-models-ts.git"
+  },
+  "homepage": "https://github.com/tommy351/kubernetes-models-ts",
+  "author": "Rueian <rueiancsie@gmail.com>",
```
Comment (tommy351) [inline packages/sealed-secrets/package.json:10]: Please change this to your name.

**Comment by douglasduteil** [comment]
Sorry for the copy paste typos @tommy351

**Comment by douglasduteil** [comment]
Thanks for the release @tommy351

**Pull request merged**